packages/@aws-cdk/aws-pipes-targets-alpha/lib/cloudwatch-logs.ts

```diff
@@ -0,0 +1,88 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import { ILogGroup } from 'aws-cdk-lib/aws-logs';
+
+/**
+ * CloudWatch Logs target properties.
+ */
+export interface CloudWatchLogsTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default - none
+   */
+  readonly inputTransformation?: IInputTransformation;
+
+  /**
+   * The name of the log stream.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetcloudwatchlogsparameters.html#cfn-pipes-pipe-pipetargetcloudwatchlogsparameters-logstreamname
+   * @default - none
+   */
+  readonly logStreamName?: string;
+
+  /**
+   * The time the event occurred, expressed as the number of milliseconds after Jan 1, 1970 00:00:00 UTC.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetcloudwatchlogsparameters.html#cfn-pipes-pipe-pipetargetcloudwatchlogsparameters-timestamp
+   * @default - none
+   */
+  readonly timestamp?: string;
```

Comment:
Any reason not to have a `number` here? You can convert it back to a string in the constructor, and it reduces the chance that a user enters something like a ISO date

Reply:
I can't get this to work in the Console even:

```
[ECMA 262 regex "^\$(\.[\w/_-]+(\[(\d+|\*)\])*)*$" does not match input string "1719416220650"]
```

Reply:
I'm no regex expert but I can't make sense of that 🤔

Reply:
@redwheeler3 is there a disconnect between the docs and the API? or am I missing something?

Reply:
For this one, we expect a JSONPath expression that references the timestamp in your payload. If you try the string "$.data.timestamp" it will validate.

Reply:
Leaving this blank defaults to the current time.

Reply:
@redwheeler3 I updated the above and added an example. I also confirmed `$.data.timestamp` worked in the Console. We should update the [API docs](https://docs.aws.amazon.com/eventbridge/latest/pipes-reference/API_PipeTargetCloudWatchLogsParameters.html).